compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUTensorCoreVectorization.cpp

```diff
@@ -104,11 +104,11 @@ static Optional<SmallVector<int64_t, 4>> getGPUTCNativeVectorSize(
   // Currently hardcode the size of wmma operation. When more cases are
```

Comment:
`getGPUTCNativeVectorSize` is really the size we want to operate unit of math and load on. This is not really the native size of the math instruction.